src/compiler.jl

```diff
@@ -990,27 +987,27 @@ end
 function get_array_elsz(B, array)
     ctx = LLVM.context(array)
     ST = get_array_struct(ctx)
-    elsz = LLVM.IntType(16; ctx)
+    elsz = LLVM.IntType(16)
     array = LLVM.pointercast!(B, array, LLVM.PointerType(ST, LLVM.addrspace(LLVM.value_type(array))))
-    v = inbounds_gep!(B, ST, array, LLVM.Value[LLVM.ConstantInt(Int32(0); ctx), LLVM.ConstantInt(Int32(3); ctx)])
+    v = inbounds_gep!(B, ST, array, LLVM.Value[LLVM.ConstantInt(Int32(0)), LLVM.ConstantInt(Int32(3))])
     return LLVM.load!(B, elsz, v)
 end
 
 function get_array_len(B, array)
     ctx = LLVM.context(array)
     ST = get_array_struct(ctx)
     array = LLVM.pointercast!(B, array, LLVM.PointerType(ST, LLVM.addrspace(LLVM.value_type(array))))
-    v = inbounds_gep!(B, ST, array, LLVM.Value[LLVM.ConstantInt(Int32(0); ctx), LLVM.ConstantInt(Int32(1); ctx)])
-    sizeT = LLVM.IntType(8*sizeof(Csize_t); ctx)
+    v = inbounds_gep!(B, ST, array, LLVM.Value[LLVM.ConstantInt(Int32(0)), LLVM.ConstantInt(Int32(1))])
+    sizeT = LLVM.IntType(8*sizeof(Csize_t))
     return LLVM.load!(B, sizeT, v)
 end
 
 function get_array_nrows(B, array)
     ctx = LLVM.context(array)
```

Comment:
can you remove the other ctx's that aren't used, like here? Just search for `context(` and that should probably find most of them